oauth2_http/javatests/com/google/auth/oauth2/MockMetadataServerTransport.java

```diff
@@ -145,6 +155,61 @@ public LowLevelHttpResponse execute() throws IOException {
               .setContent(signature);
         }
       };
+    } else if (isIdentityDocumentUrl(url)) {
+      if (idToken != null) {
+        return new MockLowLevelHttpRequest(url) {
+          @Override
+          public LowLevelHttpResponse execute() throws IOException {
+            return new MockLowLevelHttpResponse().setContent(idToken);
+          }
+        };
+      }
+
+      // https://cloud.google.com/compute/docs/instances/verifying-instance-identity#token_format
+      URL u = new URL(url);
+      Map<String, String> query_pairs = new HashMap<String, String>();
```

Comment:
queryPairs per Google style

Reply:
done...

side note: i found it suprising parsing queryparameters was so manual....